fictadvisor-api/src/v2/api/controllers/PageTextController.ts

```diff
@@ -45,66 +33,26 @@ export class PageTextController {
     return this.pageTextService.getAll(query);
   }
 
-  @ApiCookieAuth()
-  @ApiOkResponse({
-    type: PageTextResponse,
-  })
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidBodyException:
-      Key can not be empty
-      Value can not be empty`,
-  })
-  @ApiForbiddenResponse({
-    description: `\n
-    NoPermissionException:
-      You do not have permission to perform this action`,
-  })
-  @ApiUnauthorizedResponse({
-    description: `\n
-    UnauthorizedException:
-      Unauthorized`,
-  })
   @ApiEndpoint({
     summary: 'Create text for page',
+    documentation: PageTextDocumentation.CREATE,
     permissions: PERMISSION.PAGE_TEXTS_CREATE,
   })
   @Post()
   create (
-    @Body() body: CreatePageTextDTO,
+    @Body(PageTextByKeyPipe) body: CreatePageTextDTO,
   ) {
     return this.pageTextService.create(body);
   }
 
-  @ApiCookieAuth()
-  @ApiOkResponse({
-    type: PageTextsResponse,
-  })
-  @ApiBadRequestResponse({
-    description: `\n
-    InvalidBodyException:
-      Key can not be empty
-      
-    InvalidEntityId:
-      Resource with such id is not found`,
-  })
-  @ApiForbiddenResponse({
-    description: `\n
-    NoPermissionException:
-      You do not have permission to perform this action`,
-  })
-  @ApiUnauthorizedResponse({
-    description: `\n
-    UnauthorizedException:
-      Unauthorized`,
-  })
   @ApiEndpoint({
     summary: 'Update many page texts by given keys',
+    documentation: PageTextDocumentation.UPDATE_MANY,
     permissions: PERMISSION.PAGE_TEXTS_UPDATE,
   })
   @Patch()
   async updateMany (
-    @Body() body: UpdatePageTextsDTO,
+    @Body(ValidatePageTextsPipe) body: UpdatePageTextsDTO,
   ) {
```

Comment:
we must pass an array in this pipe, but our request is an object, so we need to pass a value of `pageTexts` property 
```suggestion
    @Body('pageTexts', ValidatePageTextsPipe) body: UpdatePageTextsDTO,
  ) {
```